devnotes/pnor_layout.md

```diff
@@ -0,0 +1,161 @@
+| [HBBL](#HBBL)               | 15 | 03971000:03977FFF  | 7000    | d    | -     | same as size | _unknown_                   |
+| ATTR_TMP                    | 16 | 03978000:0397FFFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| ATTR_PERM                   | 17 | 03980000:03987FFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| [IMA_CATALOG](#IMA_CATALOG) | 18 | 03989000:039C8FFF  | 40000   | d    | -     | same as size | _unknown_                   |
+| RINGOVD                     | 19 | 039C9000:039E8FFF  | 20000   | d    | -     | same as size | _unknown_                   |
+| WOFDATA                     | 1A | 039E9000:03CE8FFF  | 300000  | d    | -     | same as size | _unknown_                   |
+| HB_VOLATILE                 | 1B | 03CE9000:03CEDFFF  | 5000    | d    | -     | same as size | _unknown_                   |
+| MEMD                        | 1C | 03cee000:03CFBFFF  | E000    | d    | -     | same as size | _unknown_                   |
+| SBKT                        | 1D | 03d02000:03D05FFF  | 4000    | d    | -     | same as size | _unknown_                   |
+| HDAT                        | 1E | 03d06000:03D0DFFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| UVISOR                      | 1F | 03d10000:03E0FFFF  | 100000  | d    | -     | same as size | _unknown_                   |
+| [BOOTKERNFW](#BOOTKERNFW)   | 20 | 03e10000:03FEFFFF  | 1E0000  | d    | -     | same as size | _unknown_                   |
+| [BACKUP_PART](#BACKUP_PART) | 21 | 03ff7000:03FFEFFF  | 8000    | d    | -     | 0            | _unknown_                   |
+
+Type:
+* l = FFS_TYPE_LOGICAL
+* d = FFS_TYPE_DATA
+* p = FFS_TYPE_PARTITION
+
+Flags:
+* b = FFS_FLAGS_U_BOOT_ENV
+* p = FFS_FLAGS_PROTECTED
+
+# Reproducibility
+Reproducibility of partitions can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Firmware/Reproducible_Builds/Status).
+
+# Partition description
+## part
+This partition contains partition table.
+
+## HBEL
+This partition is used by `hostboot` to to save crash information for later
+debug.
+More informations can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Hostboot_Debug_Howto).
+
+## GUARD
+This partition is responsible for holding information about components that
+are detected as broken.
+More information about it can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Troubleshooting/Guard_Partition).
+
+## SECBOOT
+This partition is used to hold secure variables across renppts in
+tamper-resistant manner. Because writes to PNOR can't be prevented, hash of
+this partition is stored in the `TPM NV`, and verified at the boot time.
+More informations can be found at the
+[wiki page](https://open-power.github.io/skiboot/doc/secvar/secboot_tpm.html).
+
+## HBB
+This partition is overwritten with `coreboot` image containing `bootblock`.
```

Comment:
@IgorBagnucki But what was the original usage? These sections are a mix of original purpose vs coreboot purpose. Maybe clearly define subsections in each of them and then describe what was the original purpose and what is the purpose in the coreboot case.

Reply:
I have moved this information to the table with partition layout and left here informations about original usage only